Cargo.toml

```diff
@@ -1,2 +1,14 @@
 [workspace]
 members = ["crates/*"]
+
+
+# The settings below will be applied to all crates in the workspace
+[workspace.package]
+# MSRV (minimal supported Rust version).
+# `1.64` is the version that stabilized `IntoFuture`.
```

Comment:
Is this really needed? I mean noting `IntoFuture` in the comment.

Reply:
:shrug:

Reply:
Should I remove it? I've added this comment because I thought that this may clear up why we set this exact value, but if you think it's unimportant, we can just don't add it.

Reply:
Yeah I'd rather remove it.

Reply:
Done